### ISSUE TITLE
Armature Jittering when parented

### ISSUE THREAD

**Issue by IzaZed** [UPBGE/upbge#1172]
Hi, I can't seem to get myself working with armatures, they start to jitter once the armature object is parented to a moving one

Here's the test file:

[test.zip](https://github.com/UPBGE/upbge/files/4461344/test.zip)

move around with WASD and the hammer of the gun should start to jitter, I had the same thing when working with an animated character

Is this a bug or am I missing something?

**Comment by youle31** [comment]
Hi, do you have a test file not using node logic addon?

(I fixed a bug related to play armatures actions when the armatures are parented to a camera. Blender code is using a function called ignore_parent_tx iirc to avoid this jittering and I picked this code to solve the issue)

https://github.com/UPBGE/upbge/commit/131e83e37571b91a4bbd9492c638708105bbda68

**Comment by IzaZed** [comment]
A to move forward, Left Mouse to trigger the shooting animation

Shoot at least once, apparently the jittering starts when some bone is not in its original position

[test.zip](https://github.com/UPBGE/upbge/files/4502912/test.zip)

**Comment by RafaelNasci** [comment]
Same problem here.. Can't use any armature parented to capsule in UPBGE 0.3. This was done with suzanne and 3 bones.   When I move the capsule with WASD keys, the mesh jumps out of place and the armature to another place. I have no colision on mesh nor on the armature and I tested the capsule with every physics types(box,charater, etc.) and colision boxes. Even without colision box the armature jumps. I've parented with every type possible(with automatics weights, envelops, no vertex groups etc. and still the same. Even if I apply the armature to the mesh leaving it without modifiers, it still jumps. if I press P and move to left and press ESQ to get out of game, the dashed parent line moves the same amount in the opposite direction, and the mesh and armature goes to different "random" places. Initially I've tried with a makehuman mesh with armature and everything worked very nice, I can walk, trigger different animations, etc. But when I parent the armature to the capsule all hell brakes loose.
